frontend/ui/data/keyboardlayouts/keypopup/en_popup.lua

```diff
@@ -360,11 +364,11 @@ return {
         "S",
         north = "ẞ", -- uppercase eszett
         northeast = "Ś",
-        northwest = "ʃ", -- esh, voiceless palato-alveolar fricative IPA
+        northwest = "Ș", -- S with comma below (Romanian)
         east = "Ŝ",
         west = "Š",
         south = "Ş",
-        southeast = "$",
```

Comment:
Duplicate in which way?

Reply:
The $ sign is already in the en_keyboard.lua at line 83 (second row, column 3). So I see no reason to add it here again.

Reply:
Those old layers could stand a bit more attention. ;-)

It's a fairly standard practice, fwiw.